src/main/kotlin/no/nav/medlemskap/routes/EvalueringRoute.kt

```diff
@@ -159,6 +159,9 @@ private fun loggResponse(fnr: String, response: Response) {
         kv("AaRegUtenlandsoppholdLandkode", response.datagrunnlag.gyldigeAaRegUtenlandsopphold()),
         kv("AaRegUtenlandsoppsholdPeriodeFom", response.datagrunnlag.gyldigeAaRegUtenlandsoppholdPeriodeFom().toString()),
         kv("AaRegUtenlandsoppsholdPeriodeTom", response.datagrunnlag.gyldigeAaRegUtenlandsoppholdPeriodeTom().toString()),
+        kv("skipsregister", response.datagrunnlag.gyldigSkipsregister()),
+        kv("skipstype", response.datagrunnlag.gyldigSkipstype()),
+        kv("fartsområde", response.datagrunnlag.gyldigFartsområde()),
```

Comment:
Hvis det er kombinasjonen av verdier som er interessant å se på, så foreslår jeg at du også logger en String-verdi som er sammensatt av skipsregister + skipstype + fartsområde.

Reply:
Godt poeng

Reply:
Jeg foreslår kortere navn på feltnavn, for eksempel skipsinfo.